articles/quantum-concepts-4-Qubit.md

```diff
@@ -109,7 +109,9 @@ While the previous constitute the most popular primitive gates for describing op
 The simplest such primitive is the single qubit-rotation. Three single-qubit rotations are typically considered: $R_x$, $R_y$ and $R_z$. To visualize the action of the rotation $R_x(\theta)$, for example, imagine pointing your right thumb along the direction of the $x$-axis of the Bloch sphere and rotating the vector with your hand through an angle of $\theta/2$ radians. This confusing factor of $2$ arises from the fact that orthogonal vectors are $180^\circ$ apart when plotted on the Bloch sphere, yet are actually $90^\circ$ degrees apart geometrically. The corresponding unitary matrices are:
 
 $$
```

Comment:
Since this is now a multiline equation, would you be willing to enclose it in an `align` environment and to add alignment characters? Note that unlike in LaTeX, the `$$` is still needed to begin MathJax processing.
```suggestion
$$
\begin{align}
```